src/sentry/snuba/outcomes.py

```diff
@@ -191,10 +197,9 @@ def map_row(self, row: MutableMapping[str, Any]) -> None:
 
 def get_filter(
     query: QueryDict, params: Mapping[Any, Any]
-) -> Tuple[List[Condition], Dict[str, Any]]:
+) -> Tuple[List[Conditions], Dict[str, Any]]:
```

Comment:
why the rename `Condition` -> `Conditions`?

Reply:
I renamed it since i needed to import Condition from snuba_sdk.conditions for the snql query and there would be an overlap